sky/setup_files/setup.py

```diff
@@ -64,9 +64,9 @@ def parse_readme(readme: str) -> str:
 
 install_requires = [
     'wheel',
-    # NOTE: ray 2.0.1 requires click<=8.0.4,>=7.0; We disable the
+    # NOTE: ray 2.2.0 requires click>=7.0; We disable the
```

Comment:
Local ray versions may be older than 2.2. Does that mean the `click<=8.0.4` constraint is still needed?

Reply:
I decided to upgrade the local ray version to `2.2.0` due to a bunch of the conflicts of the dependencies in #1734 . Wdyt?